addons/website/views/snippets/s_searchbar.xml

```diff
@@ -13,7 +13,7 @@
     </t>
 </template>
 <template id="s_searchbar" name="Search">
-    <section class="s_searchbar bg-200 pt48 pb48">
+    <section class="s_searchbar o_colored_level o_cc o_cc3 pt48 pb48">
```

Comment:
I don't think it's a good idea to replace the gray by a purple :grimacing: Should not we use o_cc2, in most color palettes, it should a light gray / light version of the primary color by default which feels more close to what designers chose for this snippet (bg-200)